shopify.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# If we're not already in a shim environment but still tried to set up a shim, print out the re-installation warning
+type __shopify_cli__ > /dev/null 2>&1
+if [ "$?" != "0" ]; then
+  echo "The Shopify CLI is no longer available as a Git repository. It is now delivered as a package.
+    Please visit https://shopify.github.io/shopify-app-cli for installation instructions."
+fi
```

Comment:
Is there some way that we can run this and just automatically update them?

Reply:
We'd have to:
* Detect where the user set up (.profile, .bashrc, .zshrc, etc.) the shell shim and remove it
** This might be tricky as there are a number of ways in which the user may have done this
* Remove the `~/.shopify-app-cli` folder
* Run the gem install command
** This means we would have to automatically pick the ruby version - not necessarily a problem, but we're kind of taking away the choice

So yes, it is doable, but I feel like there are a couple of assumptions around this.